src/DssSpell.sol

```diff
@@ -40,56 +43,34 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    uint256 constant ZERO_FIVE_PCT_RATE         = 1000000000158153903837946257;
-    uint256 constant ONE_SEVENTY_FIVE_PCT_RATE  = 1000000000550121712943459312;
-    uint256 constant THREE_TWENTY_FIVE_PCT_RATE = 1000000001014175731521720677;
-
-    uint256 internal constant MILLION = 10 ** 6;
-    // uint256 internal constant RAY     = 10 ** 27;
-    // uint256 internal constant WAD     = 10 ** 18;
-
-    function actions() public override {
-
-        // ---- New Aave v2 D3M ----
-        // https://vote.makerdao.com/polling/QmUMyywc#poll-detail
-        // Note: only on mainnet
-
-        // ---- MOMC Parameter Changes ----
-        // https://vote.makerdao.com/polling/QmUMyywc#poll-detail
-
-        // Stability Fee Changes
-
-        //Increase WSTETH-B Stability Fee to 0.5%
-        DssExecLib.setIlkStabilityFee("WSTETH-B", ZERO_FIVE_PCT_RATE, true);
+    uint256 constant ZERO_SEVENTY_FIVE_PCT_RATE = 1000000000236936036262880196;
+    uint256 constant ONE_PCT_RATE               = 1000000000315522921573372069;
+    uint256 constant ONE_FIVE_PCT_RATE          = 1000000000472114805215157978;
 
-        // Reduce RETH-A Stability Fee to 0.5%
-        DssExecLib.setIlkStabilityFee("RETH-A", ZERO_FIVE_PCT_RATE, true);
+    uint256 constant MILLION = 10 ** 6;
+    uint256 constant RAY     = 10 ** 27;
 
-        // Reduce WBTC-A Stability Fee to 1.75%
-        DssExecLib.setIlkStabilityFee("WBTC-A", ONE_SEVENTY_FIVE_PCT_RATE, true);
+    address immutable MCD_SPOT = DssExecLib.spotter();
 
-        // Reduce WBTC-B Stability Fee to 3.25%
-        DssExecLib.setIlkStabilityFee("WBTC-B", THREE_TWENTY_FIVE_PCT_RATE, true);
-
-        // line changes
-
-        // Increase CRVV1ETHSTETH-A line to 100 million DAI
-        // Note: only on mainnet
-
-        // Increase RETH-A line to 10 million DAI
-        DssExecLib.setIlkAutoLineDebtCeiling("RETH-A", 10 * MILLION);
-
-        // Increase MATIC-A line to 15 million DAI
-        DssExecLib.setIlkAutoLineDebtCeiling("MATIC-A", 15 * MILLION);
-
-        // Increase DIRECT-COMPV2-DAI line to 30 million DAI
-        // Note: only on mainnet
-
-        // ---- SF-001 Contributor Vesting ----
-        // Note: only on mainnet
-
-        // Bump changelog
-        DssExecLib.setChangelogVersion("1.14.9");
+    function actions() public override {
+        // Stablecoin vault offboarding
+        // https://vote.makerdao.com/polling/QmemXoCi#poll-detail
+        Fileable(MCD_SPOT).file("USDC-A",   "mat", 15 * RAY); // 1500% collateralization ratio
+        Fileable(MCD_SPOT).file("PAXUSD-A", "mat", 15 * RAY);
+        Fileable(MCD_SPOT).file("GUSD-A",   "mat", 15 * RAY);
```

Comment:
Going to check explicitly later but I'm assuming the other parameters mentioned in the poll already have the specified values?

Reply:
Confirmed, `line`, `chop`, and `tip` all have the values specified in the poll already, at least on Goerli.